002-simple login page/SimpleLoginPage.java

```diff
@@ -0,0 +1,44 @@
+package programs;
+
+import java.time.Duration;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+    public class SimpleLoginPage {

```

Comment:
It's always better to write comments on what is program intended to- just one line explaining about it